test/Factories.hs

```diff
@@ -37,8 +37,8 @@ createUser ident = do
     Entity planId _ <- createFreePlan
 
     insertEntity User
-        { userName = "John Smith (" ++ ident ++ ")"
-        , userEmail = "john-" ++ ident ++ "@gmail.com"
+        { userName = "Dummy Login"
+        , userEmail = "dummy@example.com"
```

Comment:
Yeah. I think there is something funny about the data coupling here, but I think this is good to merge. Once we've played around more with testing and authentication, we may think of something better.